### PR TITLE
Now works with webpack 2.x

### PR DESCRIPTION
**Pull request by jrrbru**

### CODE REVIEW

**Comment by jas7457** [comment]
Your readme says "This plugin uses its built-in loader to extract text from files that end in 'i81n.json' and export them into the destination directory, collated by locale."

This is no longer the case, is it?

**Comment by jrrbru** [comment]
The default file extension has changed, if that is what you are saying.

File: example/webpack.config.js [hunk 1]
```diff
@@ -8,29 +8,33 @@ var i18n = new i18nPlugin( {
...
 			{
-				test: /\.i18n\.json$/,
-				loader: i18n.loader()
+				test: /\.i18n$/,
+				use: i18n.loader( {
+					root: 'example/src'
+				} )
 			}
-		]
+		],
```
Comment (jas7457) [inline example/webpack.config.js]: Why do you need the comma here?

**Comment by jas7457** [comment]
Yes, that's all I meant

File: example/webpack.config.js (hunk 1 above)
Comment (jrrbru) [inline example/webpack.config.js]: typo

**Pull request merged**